gui/src/app/pages/HomePage/HomePage.tsx

```diff
@@ -193,25 +216,34 @@ const LowerRightView: FunctionComponent<LowerRightViewProps> = ({ width, height,
             return undefined
         }
     }, [dataFileContent])
-    const runPanelHeight = 80
+    const samplingOptsPanelHeight = 140
+    const samplingOptsPanelWidth = Math.min(150, width / 2)
```

Comment:
This might be why I'm getting panel overlap from I think the SamplerOutputView--probably better to let the browser lay the page out rather than setting a fixed height.
Given the sizes at which these UI components are rendering, it would also not be unreasonable to have them horizontally between the data.json panel and the "run sampling" & "cancel" buttons.

Alternatively, we could have the entire sampler opts be a slide-in panel.

Reply:
Yeah, we can explore different layout options in a different PR.